sdks/java/harness/src/main/java/org/apache/beam/fn/harness/data/BeamFnDataClient.java

```diff
@@ -57,7 +57,7 @@
    *
```

Comment:
update comment or consider naming the object FnDataConsumer

Reply:
Updated